web/server/codechecker_server/api/report_server.py

```diff
@@ -2568,12 +2568,14 @@ def __store_source_files(self, source_root, filename_to_hash,
         file_path_to_id = {}
 
         for file_name, file_hash in filename_to_hash.items():
+            trimmed_file_path = util.trim_path_prefixes(file_name,
```

Comment:
Why did you move trimming the file path up in the code? This way the `source_file_name` variable will refer to a file which doesn't exist.

Reply:
considering that drive is part of the absolute path, most people would assume that path prefix that should be trimmed also includes drive .. Say, i have my jenkins checks out the code to f:\jenkins\workspace\jobname\my_repo so moving the trim here made the generated zip file work and my expectation that im stripping from full path, not from "path that might reside on any drive on the box  .. did it cause things to fail somewhere ? at least things worked perfectly on windows - have about 200 runs on jenkins with source files uploaded properly..

I did change the last patch thought  but im pretty sure if thats how its needed - i need to fix the same usecase somewhere else ..